rakefile.rb

```diff
@@ -37,6 +37,12 @@
     asm.output_file = SHARED_ASSEMBLY_INFO
 end
 
+desc "Restore NuGet packages"
+exec :nuget_restore do |cmd|
+   cmd.command = "tools/nuget/NuGet.exe"

```

Comment:
Change that to `dependencies/Nancy/tools/nuget/NuGet.exe` and kill the local `tools/nuget` folder, no other repo maintains its own version of the executable